src/Symfony/Bridge/Twig/AppVariable.php

```diff
@@ -62,6 +64,14 @@ public function setDebug(bool $debug)
         $this->debug = $debug;
     }
 
+    /**
+     * @return void
```

Comment:
As this is a new method, you should add the native return type directly, as there is no BC impact forgetting to add it in a minor version (btw, that's why the CI fails)